### PR TITLE
Wrote e2e tests

### PR DESCRIPTION
**Pull request by trentonf96**
Was able to get the teacher's e2e test working. Also, wrote an e2e test that adds an assignment.

### CODE REVIEW

File: src/test/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# JAWSDB database properties for register back end
```
Comment (BowieRules) [inline src/test/resources/application.properties:1]: Did you get the email from git about exposing Database secret keys as well? :)

**Review by BowieRules — APPROVED** [review]
Looks good, @trentonf96  -- not one, but 2 E2E tests!!!

File: src/test/resources/application.properties (hunk 1 above)
Comment (trentonf96) [reply src/test/resources/application.properties:1]: Yes, initially it didn't occur to me that putting secret keys on a public repository might cause security issues. :)

Comment (BowieRules) [reply src/test/resources/application.properties:1]: Live and learn, right?

**Pull request merged**